### DISCUSSION TITLE
Can i add 24bit color on Sprites ？

### DISCUSSION THREAD

**Discussion by cookies-J** [Bodmer/TFT_eSPI#3099] [Q&A - Sprites]
我现在看到的最高支持 16位 RGB565 格式， 我看了源码应该就只是修改 TFT_eSprite 这个类，请问如果我修改的话需要注意些什么呢？谢谢！

**Comment by Bodmer** [comment]
That is not supported, it is something you will have to code yourself. Note that most displays are 16 bit so increased colour resolution will not be benefitial in the sprite. You can convert a 24 bit tcolour o 16 bit using the library functions
